[CS2113-W12-1] Sniff

cleoncheng2000 opened the pull request:
Sniff is an appointment tracker for veterinary services that help clinics better manage their working staff, improve efficiency and reduce human errors.

ChubbsBunns commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### User Interface (UI) - Class Implementation
+![img.png](img.png) <br>
```
This image has incorrect syntax. 
The C symbol should not be shown in such a way.
The function named should also be pre-fixed with + or - to showcase whether they are public/private.

lohjooh replied:
![image](https://user-images.githubusercontent.com/88373092/228757820-52b768c6-a066-4959-99c2-83db94d389ed.png)

Perhaps that it would be better to use the standard notation for the UML diagram. To do that you could use the notation `hide circle` and `skinparam classAttributeIconSize 0` in your .puml file.

ChubbsBunns commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+3. The class imports the **`SniffException`** class from the exception package and utilizes it in the **`showErrorMessage()`** method to display an error message to the user.
 
+### Command - Class Implementation
+![img_16.png](Updated%20Command%20UML.png) <br>
```
Good extension for command classes, this allows ease of exiting the application.

jeraldgau replied:
I am not very clear what the symbols represent in the class diagrams, is a different standard being used to represent them?

ChubbsBunns commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+5. If the file is stored in an incorrect format / has missing details, a **`SniffException`** is thrown.
+
+### Find - Find Implementation
+![img_17.png](img_17.png) <br>
```
The use of <<create>> twice from SniffTasks to SniffExcepion does feel slightly confusing.
Is this a duplicate?

Reynold-SL replied:
Perhaps you can consider making the activation bar a single one?
![image](https://user-images.githubusercontent.com/88631870/228733598-0ff049a6-925e-4f39-a662-5a1f365c8923.png)

ChubbsBunns commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+### Command - Class Implementation
+![img_16.png](Updated%20Command%20UML.png) <br>
+**Figure 2: UML Diagram of Command Class** <br>
+1. The **`Command class`** named `Command` belongs to the package functionalities.commands. The class is an abstract class that provides a basic template for implementing commands in the command-line interface. This class is designed to be extended by subclasses that implement specific commands, such as **`ListCommand`** and **`ConsultationCommand`**.
```
Good extension for command classes, this allows ease of exiting the application.

jeraldgau replied:
Agree with this part, I like the use of the abstract class and how the different commands are extended from it and separated out neatly.

ChubbsBunns commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### User Interface (UI) - Class Implementation
+![img.png](img.png) <br>
```
This image may have incorrect syntax.
Is the Class symbol supposed to be shown in such a way?
The function named should also be pre-fixed with + or - to showcase whether they are public/private.

KN-CY replied:
Agree with the above comment, seems like the class diagram could be using non-standard UML symbols/syntax.

ChubbsBunns commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+
+| Version | As a ... | I want to ...                               | So that I can ...                                        |
+|---------|----------|---------------------------------------------|----------------------------------------------------------|
+| v1.0    |new user| add new appointments                        |                                                          |
```
I understand how this is self-explanatory, but it might be good to rephrase/ go more in-depth in the "so-that-I-can" .
(e.g. I want to add new appointments so that I can be able to easily book appointments with a doctor).

ChubbsBunns commented on docs/DeveloperGuide.md:
````diff
@@ -6,33 +6,360 @@
+______________________________________________________________________
+```
+
+### Sample test cases
````
Good sample test cases with clear inputs and outputs

ChubbsBunns commented on docs/DeveloperGuide.md:
````diff
@@ -6,33 +6,360 @@
+```
+______________________________________________________________________
+ Sorry, an error was encountered! Here is the error description:
+ The consultation description is invalid!
````
Amazing depth on error handling

KN-CY replied:
Agreed, it is great that the specific behaviour that caused the error was identified.

ChubbsBunns commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+
+#### Mark/Unmark appointments
+
+1. Prerequisites: The UID of the appointment you want to remove. Use `list` or `find` to help you get the specific UID.
```
Earlier it was stated that you did not want the user to manually type in the UID string themselves. I understand that you can use list or find to assist one in finding the UID to help delete the appointment, but would it be better to allow for someone to delete an appointment by name? 
If this is implemented you could prevent duplicate appoint names from being created.

KN-CY commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+1. The **`Command class`** named `Command` belongs to the package functionalities.commands. The class is an abstract class that provides a basic template for implementing commands in the command-line interface. This class is designed to be extended by subclasses that implement specific commands, such as **`ListCommand`** and **`ConsultationCommand`**.
+2. The **`Command class`** has a boolean isExit field that can be set to true to exit the entire programme, or remain as false to continue running the programme. It has a **`isExit()`** method that sets that field to false initially. It also has a **`executeCommand()`** method that takes an instance of SniffTasks as a parameter and throws a SniffException if an error occurs during execution. Subc ...
+
+![img_12.png](img_12.png) <br>
+**Figure 3: Sequence Diagram showing the logical implementation of executeCommand() for the Consultation Command**
```
Should the methods in the sequence diagrams have brackets. Eg. `executeCommand()` instead of `executeCommand`?

jeraldgau replied:
Is underlining of the some of the methods and the << >> brackets part of the standard UML notation? It is not very clear what they represent.

lohjooh replied:
![image](https://user-images.githubusercontent.com/88373092/228760457-50773e5b-15a2-4cd9-9792-3b3a4ca3542c.png)

What does the `1.1` in the diagram represent? You should consider adding unconventional notation such as this as a note instead.

KN-CY commented on docs/DeveloperGuide.md:
````diff
@@ -6,33 +6,360 @@
+2. Test case: `list`<br>
+   Expected output: A list of all previously added appointments.
+   Example:
+```
+______________________________________________________________________
+1.  UID: C84120821A
+ Date: 2023-12-12 | Time: 19:00
+ Animal Name: Lulu | Animal Type: Cat
+ Owner Name: Jon | Contact Number: 91919191
+2.  UID: V12400172X | vaccine: Covid
+ Date: 2023-12-12 | Time: 19:00
+ Animal Name: Russ | Animal Type: Dog
+ Owner Name: Abel | Contact Number: 92929292
+3.  UID: S14837641R | Priority: HIGH
+ Animal Name: Polly | Animal Type: Hamster
+ Owner Name: Sam | Contact Number: 93939393
+ Start Date: 2023-12-12 | Start Time: 19:00
+ End Date: 2023-12-12 | End Time: 20:00
+______________________________________________________________________
+```
````
Seems like there could be a formatting error when this is viewed from the [DG website](https://ay2223s2-cs2113-w12-1.github.io/tp/DeveloperGuide.html)
![image](https://user-images.githubusercontent.com/78925549/228704074-6f1f7efe-86ff-43f1-b47b-63f822ba6a8c.png)

KN-CY commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+![img_12.png](img_12.png) <br>
+**Figure 3: Sequence Diagram showing the logical implementation of executeCommand() for the Consultation Command**
+
+![img_14.png](img_14.png) <br>
+**Figure 4: Sequence Diagram showing the logical implementation of executeCommand() for the Vaccination Command**
```
Is showing the step level such as "1.1.3.2" part of standard uml notation?

lohjooh replied:
![image](https://user-images.githubusercontent.com/88373092/228760005-02d1714f-1d3e-4595-8a66-18a606451731.png)

Is your notation for a method call correct? Perhaps you could consider adding parenthesis to the end of each method call .

KN-CY reviewed (commented):
Good job overall, but some possible non-standard notations used for UML diagrams.

Reynold-SL commented on docs/DeveloperGuide.md:
```diff
@@ -6,33 +6,360 @@
+3. If the user inputs omits any entry or adds any extra entry then an error message will be displayed. An error message will also be displayed if the input type is of the wrong the format.
+
+##### UID Generation
+![img_20.png](img_20.png)<br>
```
Perhaps you can consider using the standard notation for class diagrams
![image](https://user-images.githubusercontent.com/88631870/228733887-89a8ae76-1784-415c-8523-e78ba95315c0.png)

lohjooh commented on docs/DeveloperGuide.md:
````diff
@@ -6,33 +6,360 @@
+2. Test case: `find a/Dog`
+   Expected output: A list of all previously added appointments with the animal type.
+   Example:
+```
````
![image](https://user-images.githubusercontent.com/88373092/228761785-1b255e27-2dcd-4ae9-a8ce-1c3911a9f05a.png)

Within the DG page, there seems to be incorrectly formatted markdown code.
Consider adding newlines to try to remedy the errors.

lohjooh reviewed (commented):
Overall, very detailed and well-written DG. Do consider checking for some formatting errors though! Perhaps you may also want to consider more conventional notation for you diagrams.

Vignesh-30 committed:
Merge pull request #150 from Vignesh-30/Vignesh-addFindDate

Vignesh add find date

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Jemma8376 committed:
Updated DG and rectified unMark bug

11-Navya committed:
Merge pull request #151 from 11-Navya/master

Updated DG and rectified unMark bug

Jemma8376 committed:
Edits to mark and unmark command

Jemma8376 committed:
Solved conflicts

* commit 'd913d180b4ca8d3fbd1989a2c6e2452898508412':
  Removed unused import
  modified existing find command to also include additonal findDate for parsing and method calls
  Created findDate method in SniffTasks
  created boolean isDate method in appointment classes to check if there are any appointments for given date
  Edit imports
  Javadocs for ArchiveParser
  Adding JavaDocs for Archive classes and methods
  UI testing for Archive
  Editing UI Testing Files
  Edit imports
  Implementing Archive Functionality
  updated aboutus
  added PPP
  Update ReadME.md

# Conflicts:
#	src/main/java/functionalities/SniffTasks.java

11-Navya committed:
Merge pull request #153 from 11-Navya/master

solved conflicts

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Jemma8376 committed:
Changed duplicate Ui prints

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Jemma8376 committed:
Removed unused imports

11-Navya committed:
Merge pull request #154 from 11-Navya/master

updated mark and unmark

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Jemma8376 committed:
Fixed unmark

11-Navya committed:
Merge pull request #155 from 11-Navya/master

Fixed unmark

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Vignesh-30 committed:
added test for findCommand

Vignesh-30 committed:
added documentation for findType, findAppointment, findDate and findAnimal methods

Vignesh-30 committed:
added documentation for findCommand class

Vignesh-30 committed:
added documentation for FileParser class

Vignesh-30 committed:
added documentation for Storage class

Vignesh-30 committed:
Added newline and removed * import

Vignesh-30 committed:
Merge pull request #156 from Vignesh-30/Vignesh-AddDocsAndTests

Vignesh added documentation  for Storage, FileParser and Find commands in SniffTasks

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Vignesh-30 committed:
fixed PPP bugs

Vignesh-30 committed:
Merge pull request #157 from Vignesh-30/fixPPPbugs

Fix ppp bugs

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Vignesh-30 committed:
updated details in PPP page

Vignesh-30 committed:
Merge pull request #158 from Vignesh-30/Vignesh-updatePPP

Vignesh updated ppp

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into Vignesh-updatePPP

Vignesh-30 committed:
fix PPP link

Vignesh-30 committed:
Merge pull request #159 from Vignesh-30/Vignesh-fixPPPlink

Vignesh fix PPP link

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Jemma8376 committed:
Added documentation and made changes to mark and unmark command

Jemma8376 committed:
made changes to mark and unmark

* commit 'd6eeecb74a43911f254c57ad4af3adcf052b671d':
  fix PPP link
  updated details in PPP page
  fixed PPP bugs
  Added newline and removed * import
  added documentation for Storage class
  added documentation for FileParser class
  added documentation for findCommand class
  added documentation for findType, findAppointment, findDate and findAnimal methods
  added test for findCommand

# Conflicts:
#	src/main/java/functionalities/SniffTasks.java

Jemma8376 committed:
Changed indentation

NivethaUdayakumar committed:
Update PPP - Udayakumar Nivetha

Jemma8376 committed:
Removed unrecognised lines

NivethaUdayakumar committed:
Merge pull request #161 from NivethaUdayakumar/v.2.1

Update PPP - Udayakumar Nivetha

11-Navya committed:
Merge pull request #160 from 11-Navya/master

Added documentation and made changes to mark and unmark

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

NivethaUdayakumar committed:
Update PPP: Udayakumar Nivetha

NivethaUdayakumar committed:
Merge pull request #162 from NivethaUdayakumar/v.2.1

Update PPP: Udayakumar Nivetha

NivethaUdayakumar committed:
update ppp links

NivethaUdayakumar committed:
Merge pull request #163 from NivethaUdayakumar/v.2.1

update ppp links

NivethaUdayakumar committed:
Edit PPP content

NivethaUdayakumar committed:
Merge pull request #164 from NivethaUdayakumar/v.2.1

Edit PPP content

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Vignesh-30 committed:
deleted unused file

Vignesh-30 committed:
Merge pull request #165 from Vignesh-30/master

deleted old PPP file

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Jemma8376 committed:
made changes to the code

Jemma8376 committed:
made changes to commands

Jemma8376 committed:
Fixed indentation

11-Navya committed:
Merge pull request #166 from 11-Navya/master

made changes to the code

cleoncheng2000 committed:
Add docs for owner and animal class

cleoncheng2000 committed:
Update parser to handle repeated command inputs

cleoncheng2000 committed:
Added javadoc for consultation, surgery and vaccination class

cleoncheng2000 committed:
Add javadoc for command class

cleoncheng2000 committed:
Add javadoc for consultation, help, surgery and vaccination commands

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Jemma8376 committed:
Added edit functionality to appointments

11-Navya committed:
Merge pull request #167 from 11-Navya/master

Added edit functionality to appointments

Jemma8376 committed:
edited UG

Jemma8376 committed:
Fixed javadocs

11-Navya committed:
Merge pull request #168 from 11-Navya/master

edited UG

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Jemma8376 committed:
Modified edit command

11-Navya committed:
Merge pull request #169 from 11-Navya/master

Modified edit command

dendendenden04 committed:
Remove assert for listAppointments method

Assertion not working as intended.

cleoncheng2000 committed:
Fix doc format warning for owner class

cleoncheng2000 committed:
Update parse adding appointments error handling

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

dendendenden04 committed:
Change the formatting for displaying appointment details

Also javadocs for Snifftasks

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

NivethaUdayakumar committed:
Bug fixes for list and archive

NivethaUdayakumar committed:
Merge pull request #170 from NivethaUdayakumar/v.2.1

Remove Asserts: V.2.1

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

cleoncheng2000 committed:
Added javadoc for add appointment and help command parsing

cleoncheng2000 committed:
Update JUnit tests names for adding appointments

cleoncheng2000 committed:
Add JUnit for add appointment parsing

Jemma8376 committed:
Added J-unit testing for mark and unmark. Added javadocs.

cleoncheng2000 committed:
Update JUnit for adding appointment parsing

11-Navya committed:
Merge pull request #171 from 11-Navya/master

Added J-unit testing for mark and unmark. Added javadocs.

cleoncheng2000 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into CleonCheng-Docs&JUnit&Bugs

# Conflicts:
#	src/main/java/functionalities/parser/Parser.java

cleoncheng2000 committed:
Merge pull request #172 from cleoncheng2000/CleonCheng-Docs&JUnit&Bugs

Update adding appintment parser, JUnit and documentation

Jemma8376 committed:
Added exception to edit command

Jemma8376 committed:
Made changes to the edit and fixed bugs

* commit 'c10bb65defc36e55c20f826e0d31a0494f7f6d71':
  Update JUnit for adding appointment parsing
  Add JUnit for add appointment parsing
  Update JUnit tests names for adding appointments
  Added javadoc for add appointment and help command parsing
  Update parse adding appointments error handling
  Fix doc format warning for owner class
  Add javadoc for consultation, help, surgery and vaccination commands
  Add javadoc for command class
  Added javadoc for consultation, surgery and vaccination class
  Update parser to handle repeated command inputs
  Add docs for owner and animal class

# Conflicts:
#	src/main/java/functionalities/parser/Parser.java

Jemma8376 committed:
fixed bugs

Jemma8376 committed:
fixed newline bug

Jemma8376 committed:
Fixed bugs

Jemma8376 committed:
Fixed indentation

Jemma8376 committed:
modified indentation

11-Navya committed:
Merge pull request #173 from 11-Navya/master

Fixed bugs and resolved conflicts

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

NivethaUdayakumar committed:
Change Naming Conventions for Archive Parser

NivethaUdayakumar committed:
Performing JUnit Testing for ArchiveParser and FileParser

NivethaUdayakumar committed:
Add newline to ArchiveParser

NivethaUdayakumar committed:
Merge pull request #174 from NivethaUdayakumar/v.2.1

JUnit Testing for ArchiveParser and FileParser

NivethaUdayakumar committed:
Update PPP links

NivethaUdayakumar committed:
Merge pull request #175 from NivethaUdayakumar/v.2.1

Update PPP links

cleoncheng2000 committed:
Update PPP link in aboutus

dendendenden04 committed:
Implement Junit test for duplicate appointment check

Updated DG, UG and Dennis PPP

dendendenden04 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into ListFormatting

# Conflicts:
#	src/main/java/functionalities/SniffTasks.java

dendendenden04 committed:
Edit junit test for duplicate checks

dendendenden04 committed:
Edit junit test for duplicate checks

dendendenden04 committed:
Remove unused imports for SniffTasksTest

dendendenden04 committed:
Merge pull request #176 from dendendenden04/ListFormatting

Formatting of appointment details

cleoncheng2000 committed:
Add features to PPP

cleoncheng2000 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into CleonCheng-UGDGPPP

# Conflicts:
#	docs/AboutUs.md

cleoncheng2000 committed:
Update UG for adding appointments.

dendendenden04 committed:
minor edits to UG and PPP

dendendenden04 committed:
Update UG remove and list features to match updated output

dendendenden04 committed:
Merge pull request #177 from dendendenden04/master

minor edits to UG and PPP

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Vignesh-30 committed:
Updated UG for find command

Vignesh-30 committed:
fix bug

Vignesh-30 committed:
Merge pull request #178 from Vignesh-30/Vignesh-UpdateUG

Vignesh update ug

dendendenden04 committed:
Update AboutUs.md to give the correct hyperlink

dendendenden04 committed:
Merge pull request #179 from dendendenden04/master

Update AboutUs.md to give the correct hyperlink

dendendenden04 committed:
Update remove parser to new format for parsing

Also update UG on expected command line messages.

cleoncheng2000 committed:
Update parser to handle duplicate commands

dendendenden04 committed:
Merge pull request #180 from dendendenden04/master

Update remove parser to new format for parsing

cleoncheng2000 committed:
Update parsing for getCommand

cleoncheng2000 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into CleonCheng-UGDGPPP

# Conflicts:
#	docs/AboutUs.md

cleoncheng2000 committed:
Update parsing in getCommand

cleoncheng2000 committed:
Merge pull request #181 from cleoncheng2000/CleonCheng-UGDGPPP

Update parser, UG and PPP

Jemma8376 committed:
Updated DG

11-Navya committed:
Merge pull request #182 from 11-Navya/master

Updated DG

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Jemma8376 committed:
Updated DG and fixed bug

Jemma8376 committed:
Merge commit '60fe5cd28bd5edcad90ab389606fb201c967e8d6'

* commit '60fe5cd28bd5edcad90ab389606fb201c967e8d6': (22 commits)
  Update parsing in getCommand
  Update parsing for getCommand
  Update parser to handle duplicate commands
  Update remove parser to new format for parsing
  Update AboutUs.md to give the correct hyperlink
  fix bug
  Updated UG for find command
  Update UG remove and list features to match updated output
  minor edits to UG and PPP
  Update UG for adding appointments.
  Add features to PPP
  Remove unused imports for SniffTasksTest
  Edit junit test for duplicate checks
  Edit junit test for duplicate checks
  Implement Junit test for duplicate appointment check
  Update PPP link in aboutus
  Update PPP links
  Add newline to ArchiveParser
  Performing JUnit Testing for ArchiveParser and FileParser
  Change Naming Conventions for Archive Parser
  ...

Jemma8376 committed:
Edited the edit command

11-Navya committed:
Merge pull request #183 from 11-Navya/master

fixed bugs and updated DG

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Vignesh-30 committed:
Updated DG

Vignesh-30 committed:
fixed typo in UG

Vignesh-30 committed:
Merge pull request #184 from Vignesh-30/Vignesh-updateDG

Vignesh update dg

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

cleoncheng2000 committed:
Update UG and DG

cleoncheng2000 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into CleonCheng-UGDGBugs

# Conflicts:
#	docs/DeveloperGuide.md

cleoncheng2000 committed:
Update help function with all commands

cleoncheng2000 committed:
Update DG manual testing with help function

cleoncheng2000 committed:
Add help function to UG

cleoncheng2000 committed:
Update help function, UG and DG

cleoncheng2000 committed:
Update parser split bug

cleoncheng2000 committed:
Merge pull request #185 from cleoncheng2000/CleonCheng-UGDGBugs

Update parser,UG,DG

Vignesh-30 committed:
Merge remote-tracking branch 'team_repo/master'

cleoncheng2000 committed:
Update aboutus and PPP

Jemma8376 committed:
Updated UG ~n

Jemma8376 committed:
Merge commit '9626dd32fb6dd63d100119074638d471ac74b0c9'

* commit '9626dd32fb6dd63d100119074638d471ac74b0c9':
  Update parser split bug
  Update help function, UG and DG
  Add help function to UG
  Update DG manual testing with help function
  Update help function with all commands
  Update UG and DG

11-Navya committed:
Merge pull request #186 from 11-Navya/master

Updated UG ~n

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Jemma8376 committed:
updated UG

11-Navya committed:
Merge pull request #187 from 11-Navya/master

updated UG

Vignesh-30 committed:
changed uID to uid

Vignesh-30 committed:
fixed parser bug

Vignesh-30 committed:
Merge pull request #188 from Vignesh-30/master

changed uID to uid

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

Vignesh-30 committed:
Fixed typo in UG

Vignesh-30 committed:
Merge pull request #189 from Vignesh-30/master

fixed typo in UG

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Jemma8376 committed:
updated ppp

11-Navya committed:
Merge pull request #190 from 11-Navya/master

updated ppp

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Vignesh-30 committed:
Added SNIFF logo

Vignesh-30 committed:
fixed typo

Vignesh-30 committed:
fixed typo in logo

Vignesh-30 committed:
fix newline

NivethaUdayakumar committed:
Add asserts

NivethaUdayakumar committed:
Update DG contents

Vignesh-30 committed:
fixing sniff logo

NivethaUdayakumar committed:
Create UML diagram for DG

Vignesh-30 committed:
fixing bugs in sniff logo

NivethaUdayakumar committed:
Change variable name

NivethaUdayakumar committed:
Merge pull request #192 from NivethaUdayakumar/v.2.1

Sniff UML Diagram and Archive Section for DG

Vignesh-30 committed:
trying to fix logo error

Vignesh-30 committed:
trying to fix sniff logo

NivethaUdayakumar committed:
Update Archive of User Guide

Vignesh-30 committed:
Changed empty spaces in SNIFF logo

Vignesh-30 committed:
fixed indentation for logo

NivethaUdayakumar committed:
Merge pull request #193 from NivethaUdayakumar/v.2.1

Update Archive of User Guide

Vignesh-30 committed:
fixed indentation of messages in expected text

NivethaUdayakumar committed:
Update PPP

Vignesh-30 committed:
Merge pull request #191 from Vignesh-30/Vignesh-AddLogo

Vignesh add SNIFF logo

Jemma8376 committed:
Edited DG

Jemma8376 committed:
Resolved conflicts

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

11-Navya committed:
Merge pull request #194 from 11-Navya/master

Resolved merge conflicts and updated DG

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

NivethaUdayakumar committed:
Update Archive Developer Guide Contents

NivethaUdayakumar committed:
Merge pull request #195 from NivethaUdayakumar/v.2.1

Update Archive Developer Guide

NivethaUdayakumar committed:
Update PPP

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

NivethaUdayakumar committed:
Merge pull request #196 from NivethaUdayakumar/v.2.1

Update PPP

Vignesh-30 committed:
Fixed typos in DG and fixed missing images

Vignesh-30 committed:
Merge pull request #197 from Vignesh-30/master

Fixed typo in DG and missing images

Vignesh-30 committed:
Fixed merge conflicts

Vignesh-30 committed:
Merge pull request #198 from Vignesh-30/master

Fixed merge conflicts

cleoncheng2000 committed:
Add addAppointment Sequence diagram and update DG

Jemma8376 committed:
Fixed edit bug

cleoncheng2000 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into CleonCheng-UGDGBugs

# Conflicts:
#	docs/DeveloperGuide.md

cleoncheng2000 committed:
Merge pull request #199 from cleoncheng2000/CleonCheng-UGDGBugs

Update DG

11-Navya committed:
Merge pull request #200 from 11-Navya/master

Fixed edit bug

Vignesh-30 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

cleoncheng2000 committed:
Update PPP

cleoncheng2000 committed:
Merge pull request #201 from cleoncheng2000/CleonCheng-UGDGBugs

Update PPP

Vignesh-30 committed:
Final update to UG

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

Vignesh-30 committed:
Merge pull request #202 from Vignesh-30/master

Fixed final UG change

NivethaUdayakumar committed:
Edit DG

NivethaUdayakumar committed:
Merge pull request #203 from NivethaUdayakumar/v.2.1

Edits to DG

Jemma8376 committed:
Made edits to UG

11-Navya committed:
Merge pull request #204 from 11-Navya/master

Made edits to UG

dendendenden04 committed:
Update breakline to quick start

NivethaUdayakumar committed:
Edits to Developer Guide

dendendenden04 committed:
Edit UG for formatting

dendendenden04 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

NivethaUdayakumar committed:
Merge pull request #205 from NivethaUdayakumar/v.2.1

Edits to Developer Guide

dendendenden04 committed:
Merge pull request #206 from dendendenden04/master

Fix UG formatting bugs

cleoncheng2000 committed:
Add help command to command summary

cleoncheng2000 committed:
Merge pull request #207 from cleoncheng2000/CleonCheng-UGDGBugs

Add help command to command summary

NivethaUdayakumar committed:
Update DG

cleoncheng2000 committed:
Add bye to command summary

NivethaUdayakumar committed:
Update DG Acknowledgements

NivethaUdayakumar committed:
Merge pull request #208 from NivethaUdayakumar/v.2.1

Update DG

cleoncheng2000 committed:
Merge pull request #209 from cleoncheng2000/CleonCheng-UGDGBugs

Add bye to command summary

dendendenden04 committed:
Remove sequence diagram for remove

dendendenden04 committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp

# Conflicts:
#	docs/DeveloperGuide.md

dendendenden04 committed:
Reformat DG intro section

dendendenden04 committed:
Add break lines in DG

Jemma8376 committed:
Edited DG

dendendenden04 committed:
Merge pull request #210 from dendendenden04/master

Reformat DG

Jemma8376 committed:
Edited DG

11-Navya committed:
Merge pull request #211 from 11-Navya/master

Edited DG

dendendenden04 committed:
Fix UG formatting

dendendenden04 committed:
Merge pull request #212 from dendendenden04/master

Fix UG formatting

cleoncheng2000 committed:
Update DG

cleoncheng2000 committed:
Merge pull request #213 from cleoncheng2000/CleonCheng-DGBugs

Update DG

cleoncheng2000 committed:
Update edit to diretly below diagrams

cleoncheng2000 committed:
Merge pull request #214 from cleoncheng2000/CleonCheng-DGBugs

Update edit on DG

cleoncheng2000 committed:
Update DG

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

cleoncheng2000 committed:
Merge pull request #215 from cleoncheng2000/CleonCheng-DGBugs

Update DG

NivethaUdayakumar committed:
Edit PPP link

NivethaUdayakumar committed:
Merge branch 'master' of https://github.com/AY2223S2-CS2113-W12-1/tp into v.2.1

NivethaUdayakumar committed:
Merge pull request #216 from NivethaUdayakumar/v.2.1

Edit PPP link - Udayakumar Nivetha
